### PR TITLE
Add test case table updates to changelog for march

### PR DESCRIPTION
**Pull request by jacksonhardy**

### CODE REVIEW

**Review by ashleeradka — APPROVED** [review]
Left a comment about format/lint + accidental copy of `Additional Headers on API Nodes`, probably because I was distracting you with a bunch of questions lol 🤓 

Thank you for walking through this process with me!

File: fern/docs/content/change-log/2024-03.mdx [hunk 1]
```diff
@@ -2,6 +2,22 @@
 title: Changelog | March, 2024
 ---
 
+## Test Suite Table Updates
+
+_March 21th, 2024_
+
+The Test Cases Table on the Test Suites page has been updated to use the same new styling and functionality as the Test Cases Table that you'll find on Evaluation Reports!
```
Comment (noanflaherty) [inline fern/docs/content/change-log/2024-03.mdx:9]:
```suggestion
The Test Cases table on the Test Suites page has been updated to use the same new styling and functionality as the Test Cases table that you'll find when viewing a Prompt/Workflow Evaluation Report. With this, adding, editing, and deleting Test Cases is generally more reliable. Additionally, special variables types, like Chat History, have an improved display are are no longer displayed as raw JSON.
```

**Review by noanflaherty — APPROVED** [review]

File: fern/docs/content/change-log/2024-03.mdx (hunk 1 above)
Comment (jacksonhardy) [reply fern/docs/content/change-log/2024-03.mdx]: applied during merge resolution

File: fern/docs/content/change-log/2024-03.mdx [hunk 2]
```diff
@@ -74,6 +74,14 @@ This is particularly helpful if you want to visualize what your Workflow looked
 
 ![Workflow Deployment Execution](https://storage.googleapis.com/vellum-public/help-docs/workflow-deployment-execution-diagram.png)
 
+## Test Suite Table Updates
+
+_March 21th, 2024_
```
Comment (ashleeradka) [inline fern/docs/content/change-log/2024-03.mdx:79]:
```suggestion
_March 21st, 2024_
```

**Review by ashleeradka — APPROVED** [review]

**Pull request merged**